.github/workflows/build.yml

```diff
@@ -139,6 +139,9 @@ jobs:
         run: conda install -y openblas pkgconfig
 
       - name: Build wheels
+        env:
+          CIBW_ARCHS_MACOS: x86_64 universal2
+          CIBW_ARCHS_LINUX: auto aarch64
```

Comment:
I'm no expert on this stuff, but it seems to be failing due to only wanting one of these to be set at a time. I think you will need to put the two architectures inside the matrix, so you can use something like {{ matrix.arch }} here, and then it will kick off 2x the jobs, one for each arch choice.